First_project/First_project/Profile/ProfileViewController.swift

```diff
@@ -9,60 +9,35 @@ import UIKit
 
 class ProfileViewController: UIViewController {
     
-//   private let areaView: UIView = {
-//        let view = UIView()
-//       view.backgroundColor = .lightGray
-//        view.translatesAutoresizingMaskIntoConstraints = false
-//        return view
-//    }()
-//
-    private let profileHeader = ProfileHeaderView()
+    private let dataSourseAdapter = StorageTableViewAdapter()
+    
+    private lazy var profileTableView: UITableView = {
+        var tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.register(PostTableViewCell.self, forCellReuseIdentifier: String(describing: PostTableViewCell.identifier))
```

Comment:
tableView.register(PostTableViewCell.self, forCellReuseIdentifier: PostTableViewCell.identifier)

PostTableViewCell.identifier is a type String already, no sense for String(describing: String)